app/assets/javascripts/shared/SharedService.js.coffee

```diff
@@ -75,7 +75,15 @@ SharedService = ($http, $state, $window, $document) ->
     'dahlia.my-applications':
       buildUrl: (state, params) -> Service._addLanguageAndParamsToUrl(params.lang, "/my-applications")
       shouldRailsRoute: (isFirstLoad) -> !isFirstLoad && $window.ACCOUNT_INFORMATION_PAGES_REACT is "true"
-
+    'dahlia.sign-in':
+      buildUrl: (state, params) -> Service._addLanguageAndParamsToUrl(params.lang, "/sign-in")
+      shouldRailsRoute: (isFirstLoad) -> !isFirstLoad && $window.ACCOUNT_INFORMATION_PAGES_REACT is "true"
+    'dahlia.create-account':
+      buildUrl: (state, params) -> Service._addLanguageAndParamsToUrl(params.lang, "/create-account")
+      shouldRailsRoute: (isFirstLoad) -> !isFirstLoad && $window.ACCOUNT_INFORMATION_PAGES_REACT is "true"
+    'dahlia.forgot-password':
+      buildUrl: (state, params) -> Service._addLanguageAndParamsToUrl(params.lang, "/forgot-password")
+      shouldRailsRoute: (isFirstLoad) -> !isFirstLoad && $window.ACCOUNT_INFORMATION_PAGES_REACT is "true"
```

Comment:
Nice! Can you change these so that they are using different env variables (also adding /reset-password) in there. I realize this wasn't super clear in the ticket, so I changed the description/AC to make that clearer

Reply:
Ah nice catch, I knew I was forgetting something. What env variable should we use? Or do we want a new one?

Reply:
Pushed those changes, having some trouble testing reset password, maybe pair quick tomorrow?